extensions/amp-sidebar/1.0/component.js

```diff
@@ -191,44 +191,49 @@ export function SidebarToolbar({
   ...rest
 }) {
   const ref = useRef();
-  const [matches, setMatches] = useState(false);
-  const [targetEl, setTargetEl] = useState(null);
+  const [target, setTarget] = useState();
 
   useEffect(() => {
-    const window = ref.current?.ownerDocument?.defaultView;
-    if (!window) {
-      return;
-    }
-
-    const mediaQueryList = window.matchMedia(mediaQueryProp);
-    const updateMatches = () => setMatches(mediaQueryList.matches);
-    mediaQueryList.addEventListener('change', updateMatches);
-    setMatches(mediaQueryList.matches);
-    return () => mediaQueryList.removeEventListener('change', updateMatches);
-  }, [mediaQueryProp]);
+    const selector = `#${CSS.escape(toolbarTarget)}`;
+    const newTarget = document.querySelector(selector);
+    setTarget(newTarget);
+  }, [toolbarTarget]);
 
   useEffect(() => {
-    const document = ref.current?.ownerDocument;
-    if (!document) {
+    const element = ref.current;
+    const doc = ref.current?.ownerDocument;
+    if (!element || !doc || !target) {
       return;
     }
 
-    const selector = `#${CSS.escape(toolbarTarget)}`;
-    const newTargetEl = document.querySelector(selector);
-    setTargetEl(newTargetEl);
-  }, [toolbarTarget]);
+    const clone = element.cloneNode(true);
+    const style = doc.createElement('style');
+    style./*OK*/ innerHTML = `
+    #${toolbarTarget} {
+        display: none;
+    }
+    @media ${mediaQueryProp} {
+      #${toolbarTarget} {
+        display: initial;
+      }
+    }`;
```

Comment:
Should we minimize white space here?

Reply:
I was on the fence since its slightly less readable, but I'm on board with minimizing.  See the latest revision: https://github.com/ampproject/amphtml/pull/34158/commits/d1c5a3eb1d7faf164c13ae9c7b13a0b2ac9dc0ed